content/id/docs/setup/production-environment/tools/kubeadm/install-kubeadm.md

```diff
@@ -0,0 +1,307 @@
+| TCP      | Inbound   | 6443*      | Kubernetes API server   | All                       |
+| TCP      | Inbound   | 2379-2380  | etcd server client API  | kube-apiserver, etcd      |
+| TCP      | Inbound   | 10250      | Kubelet API             | Self, Control plane       |
+| TCP      | Inbound   | 10251      | kube-scheduler          | Self                      |
+| TCP      | Inbound   | 10252      | kube-controller-manager | Self                      |
+
+### Node pekerja (_worker_)
+
+| Protocol | Direction | Port Range  | Purpose               | Used By                 |
+|----------|-----------|-------------|-----------------------|-------------------------|
+| TCP      | Inbound   | 10250       | Kubelet API           | Self, Control plane     |
+| TCP      | Inbound   | 30000-32767 | NodePort Services†    | All                     |
+
+† Jangkauan port bawaan untuk [Service NodePort](/docs/concepts/services-networking/service/).
+
+Angka port yang ditandai dengan * dapat diganti (_overrideable_), sehingga kamu harus memastikan port _custom_ lainnya yang kamu sediakan juga terbuka.
+
+Meskipun port etcd turut dituliskan pada Node control-plane, kamu juga bisa menghos klaster etcd-mu sendiri
+secara eksternal atau pada port _custom_.
+
+_Plugin_ jaringan Pod yang kamu gunakan (lihat di bawah) juga mungkin membutuhkan port tertentu untuk terbuka.
+Karena hal ini dapat berbeda pada setiap _plugin_ jaringan Pod, silakan lihat
+dokumentasi _plugin_ mengenai port yang dibutuhkan.
+
+## Menginstal _runtime_ 
+
+Untuk menjalankan kontainer pada Pod, Kubernetes menggunakan
+{{< glossary_tooltip term_id="container-runtime" text="container runtime" >}}.
+
+{{< tabs name="container_runtime" >}}
+{{% tab name="Linux nodes" %}}
+
+Secara bawaan, Kubernetes menggunakan
+{{< glossary_tooltip term_id="cri" text="Container Runtime Interface">}} (CRI)
+sebagai perantara dengan _container runtime_ pilihanmu.
+
+Jika kamu tidak menentukan _runtime_, kubeadm secara otomatis mencoba untuk mendeteksi
+_container runtime_ yang terinstal dengan memindai sekumpulan soket domain Unix yang umum digunakan.
+Tabel berikut menunjukkan _container runtime_ dan lokasi soketnya:
+
+{{< table caption = "Container runtime dan lokasi soketnya" >}}
+| Runtime    | Path to Unix domain socket        |
+|------------|-----------------------------------|
+| Docker     | `/var/run/docker.sock`            |
+| containerd | `/run/containerd/containerd.sock` |
+| CRI-O      | `/var/run/crio/crio.sock`         |
+{{< /table >}}
+
+<br />
+Jika ditemukan Docker dan containerd secara bersamaan, Docker akan terpilih. Hal ini diperlukan
+karena Docker 18.09 dirilis dengan containerd dan keduanya dapat ditemukan meskipun kamu
+hanya menginstal Docker.
+Jika ditemukan selain dari kedua _container runtime_ tersebut, kubeadm akan berhenti dengan kegagalan.
+
+kubelet berintegrasi dengan Docker melalui implementasi CRI `dockershim` bawaannya.
```

Comment:
might be better to start with uppercase, but since "kubelet" cannot start with capital K, we can add helper word:
```suggestion
Komponen kubelet berintegrasi dengan Docker melalui implementasi CRI `dockershim` bawaannya.
```

Reply:
Right, that goes along with Indonesian writing rules too. Fixed in 0b74a869bea33a4eb86e43a89b5284bcd244444f